src/QueuePerThreadPool.c

```diff
@@ -268,29 +272,43 @@ static void claim_work(QPTPoolThreadData_t *tw) {
     pthread_mutex_unlock(&tw->claimed_mutex);
 }
 
-#if defined(DEBUG) && defined (QPTPOOL_QUEUE_SIZE)
 static void dump_queue_size_stats(QPTPool_t *ctx, QPTPoolThreadData_t *tw) {
+    #if defined(DEBUG) && defined (QPTPOOL_QUEUE_SIZE)
```

Comment:
Why is this considered code cleanup?

Reply:
I think I prefer compiling the function itself out. There should only be one call to this function, so there shouldn't be too many extra `#ifdefs`.

Reply:
OK -- backed it out.